tasks/install_jtdx.yml

```diff
@@ -188,28 +188,69 @@
 #    set_fact: jtdx_installer="jtdx_2.2.158_amd64_ub18.deb"
 #    when: (ansible_architecture == "x86_64" or ansible_architecture == "amd64") and ansible_distribution_major_version == "18"
 
-  - name: Determine correct JTDX version for Xubuntu/Ubuntu
-    set_fact: jtdx_installer="jtdx-2.2.0-rc155_2004_amd64.deb"
-    when: ansible_architecture == "x86_64" or ansible_architecture == "amd64"
-
-  - name: Copy JTDX from local file (because automated downloads don't work reliably) (AMD64)
-    copy:
-      src: "{{ playbook_dir }}/../files/{{ jtdx_installer }}"
-      dest: "/home/{{ ham_user }}/hamradio/{{ jtdx_installer }}"
+  # - name: Determine correct JTDX version for Xubuntu/Ubuntu
```

Comment:
Ok, I believe I am setting a bad example for you because of my being impatient / laziness. We shouldn't be committing large gobs of commented out code. We can always use revision control for what it's meant to be -- a living example of what the code was before and what it is now... If we need to get older code, we can always pull it from earlier revisions...